desktop/apps/auction2/client/index.js

```diff
@@ -11,60 +11,62 @@ import { setupCommercialFilter } from './commercial_filter'
 import { data as sd } from 'sharify'
 import _ from 'underscore'
 
-const auction = new Auction(sd.AUCTION)
-const user = sd.CURRENT_USER ? new CurrentUser(sd.CURRENT_USER) : null
+export default () => {
+  const auction = new Auction(sd.AUCTION)
+  const user = sd.CURRENT_USER ? new CurrentUser(sd.CURRENT_USER) : null
 
-// If we are on the confirm-registration path then pop up a modal
-// Page is otherwise unchanged
-if (window.location.pathname.match('/confirm-registration') && user) {
-  new ConfirmRegistrationModal({ auction: auction })
-}
-
-const clock = new ClockView({
-  modelName: 'Auction',
-  model: auction,
-  el: $('.auction2-clock')
-})
-clock.start()
-
-const calendar = new AddToCalendar({
-  el: $('.auction2-callout')
-})
+  // If we are on the confirm-registration path then pop up a modal
+  // Page is otherwise unchanged
+  if (window.location.pathname.match('/confirm-registration') && user) {
+    new ConfirmRegistrationModal({ auction: auction })
+  }
 
-// render the associated auction if there is one
-if (auction.get('associated_sale')) {
-  const associatedSale = auction.get('associated_sale')
-  mountAuctionBlock({
-    relatedAuction: true,
-    sale: associatedSale
-  }, '#associated-sale')
-}
+  const clock = new ClockView({
+    modelName: 'Auction',
+    model: auction,
+    el: $('.auction2-clock')
+  })
+  clock.start()
 
-// Render my active bids if a user is present and
-// the auction is open and not in live integration mode
-if (user && sd.AUCTION && sd.AUCTION.is_open && sd.AUCTION.is_live_open === false) {
-  const myActiveBidsTemplate = require('../templates/my_active_bids.jade')
-  const activeBids = new MyActiveBids({
-    user: user,
-    el: $('.auction2-my-active-bids'),
-    template: myActiveBidsTemplate,
-    saleId: sd.AUCTION.id
+  const calendar = new AddToCalendar({
+    el: $('.auction2-callout')
   })
-  activeBids.start()
-}
 
-// Commercial filtering (redux-powered) if there are artworks
-if (sd.AUCTION && sd.AUCTION.eligible_sale_artworks_count && sd.AUCTION.eligible_sale_artworks_count > 0) {
-  setupCommercialFilter()
-}
+  // render the associated auction if there is one
+  if (auction.get('associated_sale')) {
+    const associatedSale = auction.get('associated_sale')
+    mountAuctionBlock({
+      relatedAuction: true,
+      sale: associatedSale
+    }, '#associated-sale')
+  }
```

Comment:
Super minor, but why not 

```javascript
const sale = auction.get('associated_sale')

if (sale) {
  mountAuctionBlock({
    relatedAuction: true,
    sale 
  })
}
```

(Having `relatedAuction` and `associatedSale` seems a bit confusing, like they're referring to the same different thing.)